protocol/communities/community.go

```diff
@@ -971,191 +1045,21 @@ func (o *Community) MemberIdentity() *ecdsa.PublicKey {
 }
 
 // UpdateCommunityDescription will update the community to the new community description and return a list of changes
-func (o *Community) UpdateCommunityDescription(description *protobuf.CommunityDescription, rawMessage []byte) (*CommunityChanges, error) {
+func (o *Community) UpdateCommunityDescription(description *protobuf.CommunityDescription, rawMessage []byte, allowEqualClock bool) (*CommunityChanges, error) {
 	o.mutex.Lock()
 	defer o.mutex.Unlock()
 
-	// This is done in case tags are updated and a client sends unknown tags
-	description.Tags = requests.RemoveUnknownAndDeduplicateTags(description.Tags)
-
-	err := ValidateCommunityDescription(description)
-	if err != nil {
-		return nil, err
-	}
-
 	response := o.emptyCommunityChanges()
 
-	if description.Clock <= o.config.CommunityDescription.Clock {
+	// allowEqualClock == true only if this is community event
+	if (allowEqualClock && description.Clock < o.config.CommunityDescription.Clock) ||
+		description.Clock < o.config.CommunityDescription.Clock {
```

Comment:
If I understand the logic, it should be `<=` here.
```suggestion
		description.Clock <= o.config.CommunityDescription.Clock {
```

Reply:
good catch, thank you